src/schema_generators/generators.cljc

```diff
@@ -54,11 +54,11 @@
 
 
 (defprotocol CompositeGenerator
-  (composite-generator [s params]))
+  (composite-generator [s params opts]))
```

Comment:
Can you say more about why we want to make a new parameter rather than put this in `params`?  I think these options are the same kind of thing as the existing params, maybe?

Reply:
I think if we are open to this possibility, I am ok with keeping everything on params

Reply:
OK great, let's do that, thanks!